docs/articles/documentation/index.md

```diff
@@ -6,7 +6,8 @@ inMainMenu: true
 ---
 # Documentation
 
-* [Article1](.)
-* [Article2](.)
-* [Article3](.)
-* [Article4](.)
\ No newline at end of file
+* [Getting Started](.)
+* [Test API](.)
+* [Using TestCafe](/testcafe/documentation/using-testcafe)
```

Comment:
I wonder is it possible to make navigation links relative, so we'll be able to navigate not only on the website but in GH repo as well?